### DISCUSSION TITLE
posts/makefile%E6%95%99%E5%AD%B8/

### DISCUSSION THREAD

**Discussion by giscus** [jenhaoyang/jenhaoyang.github.io#4] [Announcements]
# posts/makefile%E6%95%99%E5%AD%B8/

Makefiles用途 用來決定大型程式需要被重新編譯的部分。

https://jenhaoyang.github.io/posts/makefile%E6%95%99%E5%AD%B8/

**Comment by jenhaoyang** [comment]
aaa
